onmt/utils/ExtendedCmdLine.lua

```diff
@@ -0,0 +1,325 @@
+------------------------------------------------------------------------------------------------------------------
```

Comment:
Are these headers and separators necessary? The lines are wrapping in my editor and on GitHub compare tool.